src/components/navigation/TheNavLarge.vue

```diff
@@ -1,13 +1,13 @@
 <template>
```

Comment:
got a screenshot of what it looks like now @MightyWizard83 ?

Reply:
@bryansamuel1 This is the LARGE:
![image](https://user-images.githubusercontent.com/32196685/80636119-38eeff00-8a5d-11ea-9724-154730b78ea9.png)

Reply:
This is the SMALL:
![image](https://user-images.githubusercontent.com/32196685/80636207-58862780-8a5d-11ea-91e7-a239749cf118.png)

Reply:
P.S. I've branched from MASTER (my bad)

I've never rebased after pushing to the origin. I'm not sure how to do it.

If you want I can reimplement it and make another pull request (it's only a couple of lines of code)